src/routes/login.js

```diff
@@ -7,7 +7,7 @@ const {getAdminFromTokenOrRedirect} = require('./loginUtilities');
 
 module.exports = (fastify, opts, done) => {
     fastify.get('/login', async (request, reply) => {
-        reply.view('src/views/login');
+        return reply.view('src/views/login');
     });
```

Comment:
it does not need the async keyword
also, you can have an implicit return in JS if you have only one instruction. You need to remove the curly braces and the `;`
```js
    fastify.get('/login', (request, reply) => reply.view('src/views/login'));
```

Reply:
Huh. Actually, I need to cherry-pick dd112d38d1865ef3d539c8a4a97c0629b3f87b78 then reformat. The other two routes cause `Promise not fulfilled` errors because they are async and don't explicitly return. I believe that's why I added `return` here, but somehow didn't get the other ones straightened out.